apps/web-giddh/src/app/invoice/proforma/proforma-list.component.ts

```diff
@@ -9,31 +9,33 @@ import {
     ViewChild,
     TemplateRef
 } from '@angular/core';
-import {FormControl} from '@angular/forms';
+import { FormControl } from '@angular/forms';
 import {
     ProformaFilter,
     ProformaGetRequest,
     ProformaItem,
     ProformaResponse,
     ProformaUpdateActionRequest
 } from '../../models/api-models/proforma';
-import {select, Store} from '@ngrx/store';
-import {AppState} from '../../store';
-import {ProformaActions} from '../../actions/proforma/proforma.actions';
-import {debounceTime, distinctUntilChanged, take, takeUntil} from 'rxjs/operators';
-import {combineLatest, Observable, ReplaySubject} from 'rxjs';
+import { select, Store } from '@ngrx/store';
+import { AppState } from '../../store';
+import { ProformaActions } from '../../actions/proforma/proforma.actions';
+import { debounceTime, distinctUntilChanged, take, takeUntil } from 'rxjs/operators';
+import { combineLatest, Observable, ReplaySubject } from 'rxjs';
 import * as moment from 'moment/moment';
-import {cloneDeep, uniqBy} from '../../lodash-optimized';
-import {ModalDirective, BsModalRef, ModalOptions , BsModalService} from 'ngx-bootstrap';
-import {InvoiceFilterClassForInvoicePreview, InvoicePreviewDetailsVm} from '../../models/api-models/Invoice';
-import {InvoiceAdvanceSearchComponent} from '../preview/models/advanceSearch/invoiceAdvanceSearch.component';
-import {GIDDH_DATE_FORMAT} from '../../shared/helpers/defaultDateFormat';
-import {InvoiceSetting} from '../../models/interfaces/invoice.setting.interface';
-import {VoucherTypeEnum} from '../../models/api-models/Sales';
-import {ActivatedRoute, Router} from '@angular/router';
-import {createSelector} from "reselect";
-import {BreakpointObserver} from '@angular/cdk/layout';
-import {GeneralService} from '../../services/general.service';
+import { cloneDeep, uniqBy } from '../../lodash-optimized';
+import { ModalDirective, BsModalRef, ModalOptions, BsModalService } from 'ngx-bootstrap';
+import { InvoiceFilterClassForInvoicePreview, InvoicePreviewDetailsVm } from '../../models/api-models/Invoice';
+import { InvoiceAdvanceSearchComponent } from '../preview/models/advanceSearch/invoiceAdvanceSearch.component';
+import { GIDDH_DATE_FORMAT } from '../../shared/helpers/defaultDateFormat';
+import { InvoiceSetting } from '../../models/interfaces/invoice.setting.interface';
+import { VoucherTypeEnum } from '../../models/api-models/Sales';
+import { ActivatedRoute, Router } from '@angular/router';
+import { createSelector } from "reselect";
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { GeneralService } from '../../services/general.service';
+
+const MULTI_CURRENCY_MODULES = ['proformas', 'estimates'];
```

Comment:
Is this required as the only vouchers that are received to this component are proforma and estimates. Please check

Reply:
I used the same logic which was getting used.